monitor.py

```diff
@@ -364,6 +364,23 @@ def check_if_gather_libvirt_dir_exists(spy_link,job_type):
         return "Request timed out"
     except requests.RequestException:
         return "Error while sending request to url"
+    
+#This is to check for hypervisor error
+def check_hypervisor_error(spy_link):
+    build_log_url = constants.PROW_VIEW_URL + spy_link[8:] + '/build-log.txt'
+    try:
+        response = requests.get(build_log_url, verify=False, timeout=15)
+        hypervisor_re = re.compile('error: failed to connect to the hypervisor')
```

Comment:
Have this error line in constants like HYPERVISOR_CONNECTION_ERROR="failed to connect to the hypervisor"
```suggestion
        hypervisor_re = re.compile(constants.HYPERVISOR_CONNECTION_ERROR)
```